components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -211,11 +266,16 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
     this.saveSuccessful = false;
   }
 
-  public saveNameChange(): void {
+  public updateTeam(): void {
     this.saveSuccessful = false;
     this.saving = true;
-    const name: string = this.updateNameForm.controls.name.value.trim();
-    this.store.dispatch(new UpdateTeam({ ...this.team, name }));
+    const newName: string = this.updateNameForm.controls.name.value.trim();
```

Comment:
[style] I'd have let this be `name` and put `name` instead of `name: newName` below 🤷‍♂